shepherd_bms/src/compute.cpp

```diff
@@ -282,4 +282,29 @@ void ComputeInterface::sendChargingStatus(bool chargingStatus)
 void ComputeInterface::MCCallback(const CAN_message_t &msg)
 {
     return;
+}
+
+void ComputeInterface::sendCellTemp(uint16_t mCellTemp, uint8_t mCellID, uint16_t minCellTemp, uint8_t minCellID, uint16_t tempAvg)
```

Comment:
just tryna start enforcing this so will comment here - can you update the function parametr names to follow our styling guide. ie mCellTemp -> m_cell_temp and tempAvg -> avg_temp